modules/pipeline/pipengine/engine.go

```diff
@@ -109,7 +109,7 @@ func (engine *Engine) loadRunningPipelines() error {
 
 	// send pipeline id by interval time instead of at once
 	total := len(pipelineIDs)
-	intervalSec := time.Duration(conf.InitializeSendRunningIntervalSec())
```

Comment:
How long does it take for 600 workflows?

Reply:
5 minutes if use default config